app/src/main/java/net/gsantner/markor/format/ActionButtonBase.java

```diff
@@ -596,7 +596,7 @@ protected final boolean runCommonAction(final @StringRes int action) {
                 return true;
             }
             case R.string.abid_common_ordered_list_renumber: {
-                renumberOrderedList(TextViewUtils.getSelection(_hlEditor)[0]);
```

Comment:
So I realized that selection is a property of the editable, not the textview. This means we can simplify the call to renumber and preserve selection